renovate.json

```
@@ -1,6 +1,7 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["github>parca-dev/.github"],
+  "automergeSchedule": ["after 12am", "before 4am"],
```

If doing that, I do not think there is a reason to open PRs if they are not going to be merged

```suggestion
  "schedule": ["after 12am", "before 4am"],
```

---

Yeah makes sense.